config/config.go

```diff
@@ -19,18 +20,25 @@ type Server struct {
 	TLS       *TLS
 	SQLDriver string
 	SQLSource string
+	LogPath   string
 }
 
 func Defaults() *Server {
 	hostname, err := os.Hostname()
 	if err != nil {
 		hostname = "localhost"
 	}
+	logPath, err := os.UserHomeDir()
```

Comment:
Not sure about this default. soju is designed to be multi-user, so logging in ~ sounds weird. I'd prefer having logs disabled by default and enabled in the config file.

Reply:
Right, makes sense